trb_template.tex

```diff
@@ -1,365 +1,212 @@
-2.1.1 & Jan 2016 & Wang      & Minor modifications and uploaded to Github\\
-2.1.1 Lite & Jun 2017 & Wang & TeX-only template \\\hline
-\end{tabular}
-\end{center}
-\end{table}
-
-\subsection{Bibliography}
-The TRB bibliography style is defined in the \verb1trb.bst1 file which should be
-in your document folder. A new command is specified, \verb1\trbcite{}1 which
-will print the authors and the number of the reference in the order in which it
-is supplied. Note that \verb1\trbcite{}1 prints both the author names and the reference number, if you simply need the number of the reference, use command \verb|\cite{}|. The References section will be appended to the end of the document.
-
-It is very easy to add reference to papers programs written by
-\trbcite{Bierlaire2003} and \trbcite{Bierlaire2008} or to papers like those
-written by \trbcite{Garrow2009} and \trbcite{Koppelman2005}. You can even go
-back and refer to Biog\'eme by \trbcite{Bierlaire2008} a second time.  You can also cite a group of similar references without printing author names \cite{TRBGuide,Bierlaire2003}.  This template also groups multiple reference numbers together if there are three or more consecutive numbers \cite{Bierlaire2003,Bierlaire2008,Garrow2009,Koppelman2005}.
-
-\section{To Do's}
-There is still work to be done on this template. Currently, the word count
-feature includes text in the abstract. 
-
-There may be other important features in the template that we have not
-considered. Ideally, we would make a \verb1trb.sty1 style class that could be
-called and we would not have to expose the user to so much \TeX-ese. This could
-be forthcoming, but not for this TRB cycle. 
-
-\section{Conclusion}
-To make the document from source in a Unix-like OS using Sweave, issue the following
-commands:
-\begin{verbatim}
-R CMD SWEAVE 'trb_template.rnw'
-pdflatex --shell-escape document.tex
-bibtex document
-pdflatex --shell-escape document.tex
-pdflatex --shell-escape document.tex
-\end{verbatim}
-The \verb1--shell-escape1 option is required to access the command line for the word count. Normally this feature is disabled because it is a route of entry for malicious software. We promise that there is no such debilitating code in this document, and we encourage you to examine any scripts for suspicious code before permitting \verb1pdflatex1 from accessing your system.
-
-For R-Studio users using Sweave and .rnw files, you may enable shell escape command in the Global Options > Sweave settings.  Moreover, if your computer does not have a Perl interpreter you will need one, such as the ActivePerl, for the wordcount to work properly.
-
-\newpage
-
-\bibliographystyle{trb}
-\bibliography{trb_template}
-
-% End line numbering
-\nolinenumbers
-\end{document}
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbunofficial}
+\usepackage{graphicx}
+\usepackage[colorlinks=true,linkcolor=blue,citecolor=blue]{hyperref}
```

Comment:
I'd use \usepackage[hidelinks]{hyperref} to keep it closer to the author guidelines and the formatting model provided by TRB.